storm-server/src/main/java/org/apache/storm/scheduler/multitenant/NodePool.java

```diff
@@ -46,6 +46,12 @@ public static int slotsAvailable(NodePool[] pools) {
         return slotsAvailable;
     }
 
+    /**
+     * Get number of available slots.
+     * @return the number of nodes that are available to be taken
```

Comment:
This is `the number of slots` not `nodes`. But I am fine with it since you are just moving things around.